.github/workflows/build.yml

```diff
@@ -43,18 +47,68 @@ jobs:
     - name: Docker image tag
       id: image
       run: |
-        echo ::set-output name=tag::ghcr.io/$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]'):sha-$(echo $GITHUB_SHA | cut -c -7)
+        echo ::set-output name=tag::$CONTAINER_REGISTRY/$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]'):sha-$(echo $GITHUB_SHA | cut -c -7)
```

Comment:
Do you mind changing to full SHA as it's what has been used in all other docker images?
such as: https://github.com/DFE-Digital/apply-for-teacher-training/pkgs/container/apply-teacher-training